web-local/src/lib/components/workspace/core/WorkspaceHeader.svelte

```diff
@@ -124,11 +129,9 @@
         </svelte:fragment>
       </IconButton>
     {/if}
+
     <div class="pl-4">
       <slot name="cta" {width} />
     </div>
-    {#if showStatus}
```

Comment:
I'm finding that this icon seems to jump around quite a bit so I'm not always sure where it is between views. Can we keep the spinner on the right side? that way it remains in a consistent location for the time being. I suspect we'll probably change its location for #1620.